CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 ---------
 
+5.0.0 (unreleased)
+******************
+
+Features:
+
+* *Backwards-incompatible*: A ``JSONDecodeError`` is raised when an
+  invalid JSON payload is passed. This allows users to return a proper
+  error message for invalid JSON payloads instead of passing silently
+  (:issue:`329`). Thanks :user:`zedrdave` for reporting.
+
+Other changes:
+
+* `simplejson <https://pypi.org/project/simplejson/>`_ is now a dependency.
```

Comment:
Can you elaborate on this?

https://pypi.org/project/simplejson/

> simplejson is the externally maintained development version of the json library included with Python 2.6 and Python 3.0, but maintains backwards compatibility with Python 2.5.

I didn't test, but the README says it is included in 2.6+ and 3.0+. How come we need it in the first place?

Reply:
We need simlejson's backport of `json.JSONDecodeError`, which doesn't exist on Python 2.7.

Reply:
Ah, OK.

Do you think we could make the requirement conditional, depending on the Python version?

Or add a comment to remember to remove it when dropping Python 2.x?

No big deal otherwise.

Reply:
OK, updated the code so that simplejson is only required on Python 2.